src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,147 @@
 --%>
 <jsp:directive.include file="/WEB-INF/jsp/include.jsp"/>
 
+<link href="<c:url value="/css/feedback.css"/>" type="text/css" rel="stylesheet"/>
+
 <script src="<rs:resourceURL value="/rs/jquery/1.11.0/jquery-1.11.0.min.js"/>" type="text/javascript"></script>
+<script src="/ResourceServingWebapp/rs/jquery-migrate/1.2.1/jquery-migrate-1.2.1.min.js" type="text/javascript"></script>
```

Comment:
@timlevett 
You're rigth. moreover these links were only usefull to upgrade jQuery version, so I've put comments on them in a new commit.
Thanks